VALIDATION.md

```diff
@@ -0,0 +1,60 @@
+# STARTTLS Policy List Validation
+
+## MTA-STS
+
+In order to be preloaded onto the STARTTLS policy list, the easiest way to do and maintain this is to have a valid MTA-STS record up for a max-age of at least half a year (10886400) is sufficient to be preloaded on the STARTTLS policy list.
```

Comment:
This is a run-on sentence (it has two main verbs with no conjunction in between).